app/views/install/advanced.html

```diff
@@ -22,17 +22,20 @@
 
     <div class="nhsuk-grid-column-two-thirds">
 
-      <h1>Developer friendly install guide</h1>
+      <h1>Advanced install guide</h1>
 
-      <p class="nhsuk-lede-text">The prototype kit is built on the <a href="http://expressjs.com/" target="_blank" rel="noopener">Express<span class="nhsuk-u-visually-hidden">opens in new window</span></a> framework, and uses the <a href="https://github.com/nhsuk/nhsuk-frontend" target="_blank" rel="noopener">NHS.UK frontend<span class="nhsuk-u-visually-hidden">opens in new window</span></a>.</p>
+      <p>The prototype kit is built on the <a href="http://expressjs.com/" rel="noopener">Express</a> framework, and uses <a href="https://github.com/nhsuk/nhsuk-frontend" rel="noopener">NHS.UK frontend</a>.</p>
```

Comment:
```suggestion
      <p>The NHS prototype kit is built on the <a href="http://expressjs.com/" rel="noopener">Express</a> framework, and uses <a href="https://github.com/nhsuk/nhsuk-frontend" rel="noopener">NHS.UK frontend</a>.</p>
```

Reply:
@edwardhorsford Technically it’s called the NHS.UK prototype kit, which is a bit of a mouthful. Maybe we could just use 'kit' here?

Reply:
sounds good 👍